src/generator.py

```diff
@@ -101,44 +102,26 @@ def get_type_string(self, t, pointer=False, name=''):
     def is_vector(self, t):
         return ('vector' in t)
 
-    def capitalize(self, input_string):
-        capitalized_words = [ x[0].capitalize() + x[1:] for x in input_string.split('_')]
-        return ''.join(capitalized_words)
+    def generate(self, definition, template, globuls=None):
+        definitionDict = json.load(open(definition), object_pairs_hook=collections.OrderedDict)
```

Comment:
while it will probably work fine, using `open()` like this leaves the file opened (ppparently until is is garbage-collected).

I'd rather use the [`with open(..)`](https://docs.python-guide.org/writing/style/#read-from-a-file) syntax or at least explicitly close the file.

Reply:
done